iepy/preprocess/ner/regexp.py

```diff
@@ -0,0 +1,107 @@
+import re
+import codecs
+
+from nltk.text import TokenSearcher as NLTKTokenSearcher
+
+from iepy.preprocess.ner.base import BaseNERRunner
+
+
+class RegExpNERRunner(BaseNERRunner):
+
+    def __init__(self, label, regexp, override=False):
+        super(RegExpNERRunner, self).__init__(override=override)
+        self.label = label
+        self.regexp = regexp
+
+    def run_ner(self, doc):
+        entities = []
+        tokens = doc.tokens
+        searcher = TokenSearcher(tokens)
```

Comment:
`TokenSearcher` is implemented as a class but it is stateless in a practical sense and it is used more like a function than a class.